### ISSUE TITLE
Deleting system apps leaves icons in drawer

### ISSUE THREAD

**Issue by sschuberth** [d4rken/sdmaid-public#497]
I'm using SD Maid Pro to delete systems apps from my S6. After deleting for example the pre-installed *Excel* app, my launcher (Nova) still shows the *Excel* icon in the drawer. If I click on the icon, I get a toast saying "App isn't installed", and I'm forwarded to Play Store to install the app.

Is there a way deleting an app could delete all associated icons from the drawer, too?

**Comment by d4rken** [comment]
Hm that depends on why they are still listed. Is this somekind of app/icon cache from the launcher?
Or is the system app still installed until reboot?

If you refresh SD Maids AppControl, can you still find the app you uninstalled?
If you clear the launchers cache (data?) is the icon gone without a device restart?

**Comment by sschuberth** [comment]
I've checked the following:

- SD Maid does not list the app anymore.
- Clearing the launcher's cache did not help (I did not clear the data as I did not want to loose its settings).

*But* I noticed Nova has a "Restart Nova launcher" in its advanced settings, and doing that indeed got rid of the icons in the drawer. So it's really about Nova somehow caching them aggressively, and not SD Maid's fault, so I'm closing this.
